try
{
    
}
catch (Exception e)
{
    Console.WriteLine(e.Message);
}

enum Test { ValueA, ValueB, ValueC };